lib/phase/flatten.js

```diff
@@ -13,7 +13,7 @@ const LABEL_MATCH = 7;
  * @param  {Object}   newJob    Job to be merged into. For ex: shared settings
  * @param  {Object}   oldJob    Job to merge. For ex: individual job settings
  */
-function merge(newJob, oldJob) {
+function merge(newJob, oldJob, fromTemplate) {
```

Comment:
If it's a `Boolean`, I would strongly urge you to reconsider changing the method's parameters then